openapi/kas-fleet-manager.yaml

```diff
@@ -980,6 +980,8 @@ components:
               type: string
             bootstrap_server_host:
               type: string
+            admin_api_server_url:
+              type: string 
```

Comment:
I'll be good to add a description here as well.
```suggestion
            admin_api_server_url:
              type: string
              description: some description here about the behaviour 
```

Reply:
Thanks @machi1990, would you feel the description I've just added is sufficient?

Reply:
`description: The kafka admin server url to perform kafka admin operations e.g acl management etc. The value will be available when the Kafka has been fully provisioned i.e it reaches a 'ready' state`

Reply:
@VanillaSpoon @MikeEdgar what do you think?

Reply:
Looks good :+1:

Reply:
Thanks @MikeEdgar 

@VanillaSpoon I think we can go ahead with the proposed change.